src/lib/waku_relay/index.ts

```diff
@@ -92,86 +87,62 @@ export class WakuRelay extends GossipSub {
 
   /**
    * Send Waku message.
-   *
-   * @param {WakuMessage} message
-   * @returns {Promise<void>}
    */
-  public async send(message: WakuMessage): Promise<void> {
-    const msg = message.encode();
-    await this.publish(this.pubSubTopic, msg);
+  public async send(
```

Comment:
I am not yet happy with this interface.

Maybe it should just accept the protobuf as `Uint8Array`?

Or maybe the encoder can be passed separately? and then only need to pass `payload`, `contentTopic` and `timestamp`

Reply:
This is just aesthetics, ignoring for now.